ophyd/tests/test_areadetector.py

```diff
@@ -417,17 +439,26 @@ class MyDetector(SingleTrigger, SimDetector):
             assert Path(fn).exists()
 
 
-@pytest.mark.skipif(not os.path.exists('/data'), reason='No /data')
+@pytest.fixture
+def h5py():
+    try:
+        import h5py
+    except ImportError as ex:
+        raise pytest.skip('h5py unavailable') from ex
```

Comment:
This is a neat trick, didn't realize you could do this. Maybe I should read the pytest docs more closely.